web/tests/message_view.test.js

```diff
@@ -440,6 +451,16 @@ run_test("show_empty_narrow_message", ({mock_template}) => {
         ),
     );
 
+    // group dm with a deactivated user
+    people.deactivate(alice);
+    set_filter([["dm", me.email + "," + alice.email]]);
```

Comment:
Don't use `me.email` for a group DM; me + one other user is a weird encoding of a non-group DM.